CRM/MembershipExtras/Job/OfflineAutoRenewal/PaymentPlan.php

```diff
@@ -602,10 +602,14 @@ private function createMembership($lineItem, $priceFieldValue) {
    * Extend membership identified by given ID.
    *
    * @param int $membershipID
+   *   ID of the membership to be extended.
+   * @param array $lineItem
+   *   Data with the line item the membership belongs to.
    */
-  private function extendExistingMembership($membershipID) {
+  private function extendExistingMembership($membershipID, $lineItem) {
```

Comment:
since you only need the start_date for now, you can just pass it alone and rename the argument to $newStartDate